src/plugins/introspection.rs

```diff
@@ -15,6 +16,84 @@ impl IntrospectionPlugin {
             started: Instant::now(),
         }
     }
+
+    async fn reply_with_uptime(&self, ctx: &Arc<Context>) -> Result<()> {
+        let elapsed = self.started.elapsed();
+
+        let days = elapsed.whole_days();
+        let hours = elapsed.whole_hours() - (days * 24);
+        let minutes = elapsed.whole_minutes() - (days * 24 * 60) - (hours * 60);
+
+        let mut ret = String::new();
+
+        if days > 0 {
+            write!(ret, "{} days ", days).unwrap();
+        }
+
+        write!(ret, "{:02}:{:02}", hours, minutes).unwrap();
+
+        ctx.mention_reply(&ret[..]).await
+    }
+
+    async fn reply_with_core_uptime(&self, ctx: &Arc<Context>) -> Result<()> {
+        let info = ctx.get_core_info().await?;
+
+        let current_time = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)?
+            .as_secs();
+
+        if current_time < info.startup_timestamp {
```

Comment:
make a max skew we tolerate (like 30s), and then respond with "server started recently" if we hit this